M/MAiNGO/build_tarballs.jl

```diff
@@ -0,0 +1,66 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "MAiNGO"
+version = v"0.7.2"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://git.rwth-aachen.de/avt-svt/public/maingo.git", "252733413a29dbe5b84a4cdaf53e60e9934f372f"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd maingo/
+git remote set-url origin https://git.rwth-aachen.de/avt-svt/public/maingo.git
```

Comment:
Why?  Isn't this already the case?
```suggestion
```

Reply:
I ran into this problem before, at least in my tests locally the git remote is changed for caching.
See https://github.com/JuliaPackaging/BinaryBuilderBase.jl/issues/355